governance.md

```diff
@@ -0,0 +1,101 @@
+## Mission & Scope
```

Comment:
I'd welcome an introductory section to describe the purpose of that very document, and who should be concerned by reading it: mission of the document vs mission of the project ;-)